pkg/infoschema/infoschema.go

```diff
@@ -43,6 +43,7 @@ type InfoSchema interface {
 	SchemaByID(id int64) (*model.DBInfo, bool)
 	TableByID(id int64) (table.Table, bool)
 	AllSchemas() []*model.DBInfo
```

Comment:
Can we remove this API now?

Reply:
Still have some usages. I will remove them after this PR.